cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -47,12 +47,25 @@ fn is_bool_ty(ty: Type) -> bool {
 
 fn int_ty_is_64(ty: Type) -> bool {
     match ty {
-        types::I8 | types::I16 | types::I32 => false,
+        types::I8
+        | types::I16
+        | types::I32
+        | types::I8X16
+        | types::I16X8
+        | types::I32X4
+        | types::I64X2 => false,
         types::I64 => true,
         _ => panic!("type {} is none of I8, I16, I32 or I64", ty),
     }
 }
 
+fn ty_is_packed(ty: Type) -> bool {
+    match ty {
+        types::I8X16 | types::I16X8 | types::I32X4 | types::I64X2 => true,
```

Comment:
You could simplify this to `ty.lane_count() > 1`.

Reply:
@bjorn3  Thanks ... will update this.

Reply:
I've been using `ty.is_vector() && ty.bits() == 128` on the FP SIMD side.

Reply:
👍 Updated